server/src/domain/library/library.service.ts

```diff
@@ -658,34 +659,40 @@ export class LibraryService extends EventEmitter {
       this.assetRepository.getLibraryAssetPaths(pagination, library.id),
     );
 
+    this.logger.verbose(`Crawled asset paths paginated`);
+
     for await (const page of pagination) {
       for (const asset of page) {
         const isOffline = !crawledAssetPaths.has(asset.originalPath);
         if (isOffline && !asset.isOffline) {
           assetIdsToMarkOffline.push(asset.id);
+          this.logger.verbose(`Added to mark-offline list: ${asset.originalPath}`);
         }
 
         if (!isOffline && asset.isOffline) {
           assetIdsToMarkOnline.push(asset.id);
+          this.logger.verbose(`Added to mark-online list: ${asset.originalPath}`);
         }
 
         crawledAssetPaths.delete(asset.originalPath);
       }
     }
 
+    this.logger.verbose(`Crawled assets have been checked for online/offline status`);
+
     if (assetIdsToMarkOffline.length > 0) {
-      this.logger.debug(`Found ${assetIdsToMarkOffline.length} offline asset(s) previously marked as online`);
+      this.logger.verbose(`Found ${assetIdsToMarkOffline.length} offline asset(s) previously marked as online`);
       await this.assetRepository.updateAll(assetIdsToMarkOffline, { isOffline: true });
     }
 
     if (assetIdsToMarkOnline.length > 0) {
-      this.logger.debug(`Found ${assetIdsToMarkOnline.length} online asset(s) previously marked as offline`);
+      this.logger.verbose(`Found ${assetIdsToMarkOnline.length} online asset(s) previously marked as offline`);
       await this.assetRepository.updateAll(assetIdsToMarkOnline, { isOffline: false });
     }
```

Comment:
these are only once per job, so debug or even info would probably be fine.

Reply:
done